notebook/starter_notebook.ipynb

```diff
@@ -0,0 +1,981 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Welcome to the Intro to Machine Learning with TritonHack 2024!\n",
+    "\n",
+    "### Importing Packages\n",
+    "Import statement allow us to use library and the built-in functions within the library so we can utilize them!\\\n",
+    "First, we are going to import pacakges that is going to be the very basic for the Data Science Project!"
```

Comment:
import **packages** that **are** going to be...

Reply:
delete "the" in that is going to be **the** very basic  for the Data Science ...